src/components/TabletApp/LeftSide/LeftSide.tsx

```diff
@@ -1,17 +1,16 @@
 import React, { useEffect, useState } from "react";
-
 import { useLocation, useParams } from "react-router-dom";
```

Comment:
separate react imports from the rest of the imports, by an empty line(coding standard procedure)